tools/objinfo.ml

```diff
@@ -433,6 +433,14 @@ let dump_obj filename =
   then dump_cmxs ic
   else exit_magic_error ~expected_kind:None (Parse_error head_error)
 
+let print_version () =
+  Format.printf "ocamlobjinfo, version %s@." Sys.ocaml_version;
```

Comment:
Is there a reason to use `Format` instead of plain `Printf`?

Reply:
Only where I copied it from!

Reply:
(which was ocamlprof.ml, I think)

Reply:
In general mixing Printf and Format is a code smell because it can result in garblet output due to Format internal buffering. I guess that it is okay to have a mix of `Printf` and `Format` (which is already the case in both these files) as long as any Format use ends with an explicit flush, with no Printf in the middle. The new additions clearly respect this property.

Reply:
Fine by me!